internal/api/rbac/roles.go

```diff
@@ -450,8 +450,8 @@ func (r *rolesDatabase) List(
 		kargoRoles = append(kargoRoles, kargoRole)
 	}
 
-	sort.Slice(kargoRoles, func(i, j int) bool {
-		return kargoRoles[i].Name < kargoRoles[j].Name
+	slices.SortFunc(kargoRoles, func(a, b *rbacapi.Role) int {
+		return cmp.Compare(a.Name, b.Name)
```

Comment:
`cmp.Compare()` works, but for strings, `strings.Compare` is a more established convention.

```suggestion
		return strings.Compare(a.Name, b.Name)
```